src/lib_ccx/hardsubx.c

```diff
@@ -255,6 +260,17 @@ struct lib_hardsubx_ctx* _init_hardsubx(struct ccx_s_options *options)
 	ctx->write_format = options->write_format;
 	ctx->subs_delay = options->subs_delay;
 	ctx->cc_to_stdout = options->cc_to_stdout;
+	ctx->russian = options->russian;
```

Comment:
Probably better to create a separate context for all Russian related variables only (only if Russian mode is activated).
ctx->russian->frame_skip, ctx->russian->start_frame etc instead of just having the options in the main ctx.

Reply:
Noted